index.js

```diff
@@ -323,13 +327,17 @@ async function go() {
     console.log(finalDiff);
     console.log();
 
+    const createPR = !(noPR || await findGithubFile('PULL_REQUEST_TEMPLATE'));
+
     await respondToUserInput(
-      'Are you sure you want to create a pull request with these corrections?',
+      `Are you sure you want to ${createPR ? 'create a pull request with' : 'push and send'} these corrections?`,
```

Comment:
Any ideas for better wording?

Reply:
How about:

1. "Are you sure you want to create a pull request with these corrections?" if `createPR` is true
2. "Are you sure you want to push these changes to <branch name> on <repository/fork owner and name>?" if `createPR` is false